### DISCUSSION TITLE
question about acme.sh --cron

### DISCUSSION THREAD

**Discussion by jorgevisentini** [acmesh-official/acme.sh#5240] [General]
What does acme.sh with the **--cron** parameter actually do?

Does it try to renew the certificate or does it **first check if the certificate needs** to be renewed?

My question is why, for example, if I issue a certificate with the **--days** parameter, will acme first check if there is a need to issue it or will it try to issue the certificate without checking?

I don't know if I was clear in my question.

**Comment by neilpang** [comment]
>>  does it first check if the certificate needs to be renewed?

yes
